src/sedcli_util.c

```diff
@@ -0,0 +1,79 @@
+#include <stdio.h>
+#include <stdint.h>
+#include <termios.h>
+#include <errno.h>
+
+#include <sys/syslog.h>
+
+#include <libsed.h>
+
+#include "argp.h"
+
+static struct termios term;
+extern sedcli_printf_t sedcli_printf;
+
+static void echo_disable()
+{
+	tcgetattr(1, &term);
+	term.c_cc[VMIN] = 1;
+	term.c_lflag &= ~(ECHO | ICANON);
+	tcsetattr(1, 0, &term);
+}
+
+static void echo_enable()
+{
+	term.c_lflag |= ECHO | ICANON;
+	tcsetattr(1, 0, &term);
+}
+
+int get_password(char *pwd, uint8_t *len, int min, int max)
```

Comment:
Do any of your future features have min and max value apart from SED_MIN_KEY_LEN and SED_MAX_KEY_LEN respectively..?

Reply:
we have the **sed_key** struct defined under lib/include/libsed.h.
Wouldn't it be a better idea to just pass the **sed_key** struct address rather than getting pwd and len individually.